test/python/circuit/test_circuit_operations.py

```diff
@@ -95,11 +103,11 @@ def test_combine_circuit_fail(self):
         qc2 = QuantumCircuit(qr2)
         qcr3 = QuantumCircuit(cr1)
 
-        self.assertRaises(CircuitError, qc1.__add__, qc2)
-        self.assertRaises(CircuitError, qc1.__add__, qcr3)
+        self.assertRaises(QiskitError, qc1.__add__, qc2)
+        self.assertRaises(QiskitError, qc1.__add__, qcr3)
```

Comment:
```suggestion
        self.assertRaises(CircuitError, qc1.__add__, qc2)
        self.assertRaises(CircuitError, qc1.__add__, qcr3)
```
After #4762 , I think these should be `CircuitError`s again.